packages/neos-ui-editors/src/Image/Components/Secondary/ImageCropper/index.js

```diff
@@ -178,7 +178,7 @@ export default class ImageCropper extends PureComponent {
                     </div>
 
                     <AspectRatioDropDown
-                        placeholder="Aspect Ratio"
+                        placeholder="Choose a Aspect Ratio"
```

Comment:
"an A"

Reply:
Wait, but actually why don't we have this stuff translated??

Reply:
In the old ui it is not translated as well >.> I'll add a label in Neos.Neos.Ui